docs/UserGuide.md

```diff
@@ -322,6 +326,18 @@ Format: `exit`
 
 The data of all the internship applications is saved automatically to the hard disk whenever any changes are made, so no manual saving is necessary.
 
+<div markdown="block" class="alert alert-danger">
+
+**:information_source: Notes about the CinternS save file:**<br>
+
+* A new folder `data` will be created with the file `applicationbook.json`. This is where your data will be stored for the next time you use the app, transferring it to another device or sharing with your friend.
+
+* Please refrain from editing this save file as the file might become corrupted. When CinternS is booted, if the file is corrupted, it will reset the save file. You will NOT be able to retrieve your data!
```

Comment:
Maybe we can make this line less harsh, especially since the ability to edit the save file is technically a requirement. How about something like this?

```suggestion
* Editing the save file directly is discouraged as it might become corrupted. When CinternS is booted, if the file is corrupted, it will reset the save file. You will NOT be able to retrieve your data!
```